Makefile

```diff
@@ -3,6 +3,7 @@ SERVICE_NAME := validation-service
 LOG_LEVEL := info
 PORT := 8888
 VERSION := dev-SNAPSHOT
+HOST_DIR ?= $(shell pwd)/testdata
```

Comment:
I'm not sure of the value of `?=` over `:=` here. Running through the Makefile I think any command line overrides would be to the makefile as an arg (like LOG_LEVEL does) rather than relying on a system ENV or something.

Reply:
I think that it needs to be `?=` otherwise the path won't resolve to the new value if set by doing `export HOST_DIR` on the CLI

Reply:
Right, but I thought we were going to just use the arg for testing (i.e., Makefile use)? The ENV use is enabled for prod through the Dockerfile.